src/test/shell/bazel/bazel_symlink_test.sh

```diff
@@ -101,6 +101,22 @@ def _write_impl(ctx):
 write = rule(implementation = _write_impl, attrs = {"contents": attr.string()})
 EOF
 
+  # We use python rather than a simple ln since the latter doesn't handle dangling symlinks on
+  # Windows.
+  mkdir -p symlink_helper
+  cat > symlink_helper/BUILD <<EOF
```

Comment:
nit: 'EOF' is generally preferred unless you need variable substitution.

Reply:
Could you take care of it? I'm on mobile atm. Thanks!